db/Makefile

```diff
@@ -32,7 +32,7 @@ dmrmarc2.tmp: dmrmarc.tmp
 	awk -F, -f insert_nick.awk <$< >$@
 	
 dmrmarc.tmp:
-	python2 get_dmrmarc_json.py >$@
+  timeout 120 wget --no-check-certificate --wait=3 'https://dmr-marc.net/static/users.csv' -O $@
```

Comment:
tab vs spaces.

Reply:
Make up your mind.  Wolf bitches if you use tabs so I set my editor to expand to spaces, so now you want tabs.  I give up.

Reply:
if you dont use tabs, make would fail.